rest/src/main/java/com/carhouse/rest/config/SpringJDBCConfig.java

```diff
@@ -21,7 +18,10 @@
  * @author Katuranau Maksimilyan
  */
 @Configuration
-@PropertySource("classpath:${jdbc.properties:jdbc-connection}.properties")
+@PropertySources({
+        @PropertySource("classpath:application.properties"),
+        @PropertySource("classpath:${jdbc.properties.file}.properties"),
```

Comment:
When `integrationTest` isn't provided what will value be here?

Reply:
I have a default value for this property in my pom file and when I set integrationTest profile I override this value